src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
```

```suggestion
  padding: 32px 14px 15px;
```
You can give top padding to the card so you don't need to set a top margin for the image